en/config_mc/filter_tuning.md

```diff
@@ -35,24 +35,57 @@ Below we look at the impact of the low pass filters.
 
 ## Filters
 
-This is the filtering pipeline for the controllers in PX4:
+The filtering pipeline for the controllers in PX4 is described below.
 
-- On-chip DLPF for the gyro sensor.
-  This is disabled on all chips where it can be disabled (if not, the cutoff frequency is set to the highest level of the chip).
-- A notch filter on the gyro sensor data that is used to filter out narrow band noise, for example harmonics at the rotor blade pass frequency.
-  This filter can be configured using [IMU_GYRO_NF0_BW](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_BW) and [IMU_GYRO_NF0_FRQ](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_FRQ).
-- Low-pass filter on the gyro sensor data.
-  It can be configured with the [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
+::: info
+Sampling and filtering is always performed at the full raw sensor rate (commonly 8kHz, depending on the IMU).
+:::
+
+### On-chip DLPF for the Gyro Sensor
+
+This is disabled on all chips where it can be disabled (if not, cutoff frequency is set to the highest level of the chip).
+
+### Notch Filters
+
+Setups that have a significant lower-frequency noise spike (e.g. due to harmonics at the rotor blade pass frequency) can benefit from using the notch filter to clean the signal before it is passed to the low pass filter (these harmonics have a similar detrimental impact on motors as other sources of noise).
+
+Without the notch filter you'd have to set the low pass filter cutoff much lower (increasing the latency) in order to avoid passing this noise to the motors.
+
+#### Static Notch Filters
+
+One or two static notch filters on the gyro sensor data that are used to filter out narrow band noise, for example harmonics at the rotor blade pass frequency.
+
+The static notch filters can be configured using:
+
+- First notch filter: [IMU_GYRO_NF0_BW](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_BW) and [IMU_GYRO_NF0_FRQ](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_FRQ).
+- Second notch filter: [IMU_GYRO_NF1_BW](../advanced_config/parameter_reference.md#IMU_GYRO_NF1_BW) and [IMU_GYRO_NF1_FRQ](../advanced_config/parameter_reference.md#IMU_GYRO_NF1_FRQ).
+
+::: info
+Only two notch filters are provided.
+Airframes with more than two frequency noise spikes typically clean the first two spikes with the notch filters, and subsequent spikes using the low pass filter.
+:::
+
+#### Dynamic Notch Filters
+
+Dynamic notch filters use ESC RPM feedback and/or the onboard FFT analysis to track the rotor blade pass frequency and its harmonics.
```

Comment:
The static notch filter appears to serve the same purpose - filter blade frequency and its first harmonic (vs dynamic multiple harmonics).

1. So when would you use one or the other? 
2. The example in Daniels' original PR filtered crap from props with slight damage - I presume that results in more noise and harmonics?

Reply:
I suppose the dynamic one is generally better but only possible when you have ESC feedback. Not sure about the FFT one? 

@dagar when would you use the FFT instead of the static notch filter?

Reply:
Shall we merge and you chase him for this answer offline? My concern is that he will be too busy to respond, and we end up merging this in 2 years, no better or worse off.

Reply:
Let me ping @bresch as well, he might know and answer quicker.

Reply:
The FFT is to track a moving vibration peak (e.g.; the blade-pass frequency when ESC telemetry is not available). The static notch is usually to cover a structural mode of the airframe.
I would say that you either use the ESC feedback or the FFT to move the dynamic notches, but you can use a static notch in addition to the dynamic ones.